### PR TITLE
Allow adding a postfix to the page title

### LINKED ISSUE #6917: Page titles should include KKP (or similar)
**What is the problem, where is it and why is it an issue?**

Currently the page titles (i.e. the `<title>` tag) do not include the platform's name. If I have a tab open that just reads `Projects`, that's not very helpful. I suggest you either add "KKP" oder "Kubermatic Kubernetes Platform" as a postfix. Ideally it would be configurable (at least for the EE edition) so we can set it to our product's name instead.

**Suggest how to fix it, if possible**

Add `| Kubermatic Kubernetes Platform` to all `<title>` tags. Make the string configurable through the KubermaticConfiguration (or KubermaticSettings). If preferred, the default could also be an empty string, to keep the current UI/UX, and just allowing us to define a postfix for the title tag.

### PR DESCRIPTION
**Pull request by ahmadhamzh**
**What this PR does / why we need it**:
Added a new `postfix_page_title` property in the UI config to allow setting a custom postfix for the page title.

**Which issue(s) this PR fixes**:
Fixes #6917 

**What type of PR is this?**
/kind feature

**Does this PR introduce a user-facing change? Then add your Release Note here**:
```release-note
Added a new `postfix_page_title` property in the UI config to allow setting a custom postfix for the page title.
```

**Documentation**:
```documentation
TBD
```

### CODE REVIEW

**Comment by Waseem826** [comment]
I think it should be called postfix. Also, the format was something like `| <postfix> or Kubermatic Kubernetes Platform`.

> Add | Kubermatic Kubernetes Platform to all <title> tags. Make the string configurable through the KubermaticConfiguration (or KubermaticSettings). If preferred, the default could also be an empty string, to keep the current UI/UX, and just allowing us to define a postfix for the title tag.

**Comment by ahmadhamzh** [comment]
> If I have a tab open that just reads Projects, that's not very helpful. I suggest you either add "KKP" oder "Kubermatic Kubernetes Platform" as a postfix.

yes he mention postfix but since the issue with open tab i thought maybe it's better to be a prefix :thinking:

**Comment by Waseem826** [comment]
> > If I have a tab open that just reads Projects, that's not very helpful. I suggest you either add "KKP" oder "Kubermatic Kubernetes Platform" as a postfix.
> 
> Yes, they mentioned a postfix, but since the main concern was open tabs, I thought using a prefix might be better. 🤔

No, primary information about the page (title in this case) should come first. Secondary information like organization/platform name should come last.

File: modules/web/src/app/shared/entity/common.ts [hunk 1]
```diff
@@ -89,6 +91,8 @@ export enum ViewDisplayName {
   ClusterRestore = 'Cluster Restores',
   BackupStorageLocation = 'Backup Storage Locations',
   Access = 'Access',
+  RestAPI = 'Rest api',
```
Comment (Waseem826) [inline modules/web/src/app/shared/entity/common.ts:94]:
```suggestion
  RestAPI = 'Rest API',
```

**Review by Waseem826 — APPROVED** [review]
/approve

**Pull request merged**